src/app/containers/PageHandlers/withData.jsx

```diff
@@ -1,47 +1,65 @@
 import React from 'react';
-import { shape, element } from 'prop-types';
+import { shape, element, string } from 'prop-types';
 import articlePropTypes from '../../models/propTypes/article';
 import ErrorMain from '../ErrorMain';
-import nodeLogger from '../../lib/logger.node';
 import deepGet from '../../lib/utilities/deepGet';
+import getPassportHome from '../../lib/utilities/getPassportHome';
 
-const logger = nodeLogger(__filename);
-
-// checks for data and status keys, setting default status if not found
+// checks for data, status, setting default status if not found
 const constructRenderObject = data => ({
   status: deepGet(['status'], data) || 500,
   pageData: deepGet(['pageData'], data),
 });
 
-// checks for pageData and 200 status
-const shouldRender = data => {
+const validatePassportHome = (passportHome, service) => {
```

Comment:
You could use an implicit return here. I also think naming predicate functions with `is` at the beginning makes it more readable

```js
const isValidPassportHome = (passportHome, service) => passportHome ? passportHome === service : true
```